### ISSUE TITLE
Why I can't install tensorflow-addons by pip in windows, but can install in linux?

### ISSUE THREAD

**Issue by over-shine** [tensorflow/addons#821]
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no):
- Are you willing to maintain it going forward? (yes/no):
- Is there a relevant academic paper? (if so, where):
- Is there already an implementation in another framework? (if so, where):
- Was it part of tf.contrib? (if so, where):

**Which API type would this fall under (layer, metric, optimizer, etc.)**

**Who will benefit with this feature?**

**Any other info.**

**Comment by over-shine** [comment]
error as follows:
ERROR: Could not find a version that satisfies the requirement tensorflow-addons (from versions: none)                  ERROR: No matching distribution found for tensorflow-addons

**Comment by HbWang2017** [comment]
@over-shine  I have the same problem. So I try to download .zip package and install it without pip.
I use command:
`python XXX\addons-0.6.0\setup.py install `
to install tensorflow-addons in windoes,but have met another error.

> c1xx: fatal error C1083: Cannot open source file: 'stub.cc': No such file or directory
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2

It seems there is no way to install tensorflow-addons in windows.

**Comment by facaiy** [comment]
As https://github.com/tensorflow/addons/pull/771 has been merged into master branch, I suppose Windows support will be in the coming 0.7 release. cc @seanpmorgan Sean

**Comment by seanpmorgan** [comment]
`tfa-nightly` should be available once https://github.com/tensorflow/addons/pull/824 is merged.

But as @facaiy said the next `tensorflow-addons` 0.7 release will have windows support.

**Comment by seanpmorgan** [comment]
`pip install tfa-nightly` is now available on windows!
```
(addons-nightly) C:\Users\SeanM\Desktop> pip install tfa-nightly
(addons-nightly) C:\Users\SeanM\Desktop> python test_addons.py
2.1.0-dev20200102
0.7.0-dev
2020-01-03 09:49:49.584141: I tensorflow/core/common_runtime/eager/execute.cc:533] Executing op __inference_gelu_7 in device /job:localhost/replica:0/task:0/device:CPU:0
x: (_Arg): /job:localhost/replica:0/task:0/device:CPU:0
2020-01-03 09:49:49.593995: I tensorflow/core/common_runtime/placer.cc:114] x: (_Arg): /job:localhost/replica:0/task:0/device:CPU:0
Addons>Gelu: (Addons>Gelu): /job:localhost/replica:0/task:0/device:CPU:0
2020-01-03 09:49:49.600737: I tensorflow/core/common_runtime/placer.cc:114] Addons>Gelu: (Addons>Gelu): /job:localhost/replica:0/task:0/device:CPU:0
Identity: (Identity): /job:localhost/replica:0/task:0/device:CPU:0
2020-01-03 09:49:49.610420: I tensorflow/core/common_runtime/placer.cc:114] Identity: (Identity): /job:localhost/replica:0/task:0/device:CPU:0
identity_RetVal: (_Retval): /job:localhost/replica:0/task:0/device:CPU:0
2020-01-03 09:49:49.618120: I tensorflow/core/common_runtime/placer.cc:114] identity_RetVal: (_Retval): /job:localhost/replica:0/task:0/device:CPU:0
tf.Tensor([[ 0.345714    1.0617028  -0.11462908]], shape=(1, 3), dtype=float32)
```

**Comment by seanpmorgan** [comment]
FYI stable release for windows is out. `pip install tensorflow-addons`

**Comment by j3pic** [comment]
I'm having the same problem, except tensorflow-addons only exists on MacOS now, and doesn't exist on Linux.

**Comment by seanpmorgan** [comment]
> I'm having the same problem, except tensorflow-addons only exists on MacOS now, and doesn't exist on Linux.

Hi @j3pic TFA is available for all three OS (python 3.5-3.7):
https://pypi.org/project/tensorflow-addons/0.9.1/#files

Perhaps you need to update your pip version?

**Comment by ro-mak** [comment]
If somebody has this:
ERROR: Could not find a version that satisfies the requirement tensorflow-addons (from versions: none) ERROR: No matching distribution found for tensorflow-addons

Consider downgrading/changing your Python version, in my case I downloaded Python 3.8 which seems to me is not supported yet (1 May 2020)
